src/_pytest/pathlib.py

```diff
@@ -484,26 +484,31 @@ class ImportPathMismatchError(ImportError):
 
 
 def import_path(
-    p: Union[str, "os.PathLike[str]"],
+    path: Union[str, "os.PathLike[str]", Path],
```

Comment:
I believe `Path` is `os.PathLike[str]` so the signature change should not be needed

Reply:
Right, thanks!